docs/semgrep-ci/running-semgrep-ci-with-semgrep-app.md

```diff
@@ -89,8 +89,8 @@ To set up the CI job and connect with Semgrep Cloud Platform:
 
 1. Sign in to [Semgrep Cloud Platform](https://semgrep.dev/login). See [Signing in to Semgrep Cloud Platform](/semgrep-app/getting-started-with-semgrep-app/#signing-in-to-semgrep-app) for details on requested repository permissions and access.
 2. Click **[Projects](https://semgrep.dev/orgs/-/projects)** > **Scan New Project** > **Run Scan in CI**.
-3. Select your provider from the menu.
-4. Optional: Some providers may ask you to select your organization if applicable to your SCM tool.
+3. Select your CI provider from the menu.
+4. Optional: Some providers can ask you to select your organization if applicable to your SCM tool.
 5. Follow the steps outlined in the page:
     1. Optional: Additional permissions may be requested for Semgrep Cloud Platform to perform certain actions in your SCM tool, such as GitHub. If you prefer not to grant these permissions, Semgrep Cloud Platform provides alternative instructions in the **Don't want to install the app?** section within the page itself.
```

Comment:
@s-santillan is **Don't want to install the app?** still a valid option? I don't see it in the Platform.